contrib/command-not-found/pk-command-not-found.c

```
@@ -875,54 +997,41 @@ main (int argc, char *argv[])
-				/* TRANSLATORS: Show the user a list of packages that they can install to provide this command */
-				g_printerr ("%s:\n", _("Suitable packages are:"));
+				g_printerr ("%s\n", _("Suitable packages are:"));
 				for (i = 0; package_ids[i] != NULL; i++) {
 					parts = pk_package_id_split (package_ids[i]);
 					g_printerr ("%i\t'%s'\n", i+1, parts[PK_PACKAGE_ID_NAME]);
 					g_strfreev (parts);
 				}
-
-				/* TRANSLATORS: ask the user to choose a file to install */
 				i = pk_console_get_number (_("Please choose a package to install"), len);
 				if (i == 0) {
```

Clang (14.0.6)

[note] +4, including nesting penalty of 3, nesting level increased to 4

(clang)